ondemand.osc.edu/apps/bc_desktop/owens.yml

```diff
@@ -1,6 +1,13 @@
 ---
 title: "Owens Desktop"
 cluster: "owens"
+description: |
+  This app will launch an interactive desktop on one or more compute nodes. You
+  will have full access to the resources these nodes provide. This is analogous
+  to an interactive batch job.
+
+  If you do not need full access to the resources of a HPC node, please use the
```

Comment:
I would change 'full access to the resources of a HPC node' to 'access to the full resources of a HPC node' since it is the resources we are modifying not the access.

Reply:
Cool yea, notes I got from Eric where to be more concise anyhow. Amazon talks about cost in their descriptions, I think I may do the same.

Reply:
We can wait on adding cost to descriptions till when we merge the desktops and vdis. I'll mark this thread as resolved.